KennysWishes/app/src/main/java/com/korbkenny/kennyswishes/MainActivity.java

```diff
@@ -0,0 +1,73 @@
+package com.korbkenny.kennyswishes;
+
+import android.animation.LayoutTransition;
+import android.os.Handler;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.animation.Animation;
+import android.view.animation.AnimationUtils;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+
+public class MainActivity extends AppCompatActivity {
+
+    EditText mEditText;
+    TextView mMiddleText;
+    Button mAddWish;
+    ViewGroup mViewGroup;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        mEditText = (EditText) findViewById(R.id.edit_text);
+        mMiddleText = (TextView) findViewById(R.id.middle_text);
+        mAddWish = (Button) findViewById(R.id.add_button);
+        mViewGroup = (ViewGroup) findViewById(R.id.layout_list);
+
+        LayoutTransition l = new LayoutTransition();
+        l.enableTransitionType(LayoutTransition.CHANGING);
+        mViewGroup.setLayoutTransition(l);
```

Comment:
Since you put android:animateLayoutChanges="true" in the XML for the LinearLayout, you actually don't need to also set up the layout transition animation here. That one line in XML accomplishes the same thing.